selendroid-standalone/src/main/java/io/selendroid/standalone/server/util/HttpClientUtil.java

```diff
@@ -41,12 +41,12 @@ public static HttpClient getHttpClient() {
   }
 
   public static HttpResponse executeRequestWithPayload(String uri, int port, HttpMethod method,
```

Comment:
Wondering if we should have an overload where you don't pass the hostname that defaults to `localhost`. People might be using this in tests (which they probably shouldn't, but still..).